### DISCUSSION TITLE
Specify chapters with glob syntax?

### DISCUSSION THREAD

**Discussion by jdutant** [quarto-dev/quarto-cli#4874] [Feature Requests]
Can we specify the chapters of a book with a glob syntax? Suppose I have some conference procedeeings with 50 articles in articles/01/author.qmd, ... articles/50/authors.qmd. Looks like we can't specify them with globs:

``` yaml
chapters:
- index.qmd
- articles    # this is ignored, nothing shows up in the output
- articles/*   # ERROR: Book chapter 'articles/*' not found
- articles/*/*    # ERROR: Book chapter 'articles/*/' not found
- articles/*/*.qmd    # ERROR: Book chapter 'articles/*/*.qmd' not found
- references.qmd
```

(Context: trying to adapt quarto to journal issues and multi-author collections.)

**Comment by mcanouil** [comment]
- https://github.com/quarto-dev/quarto-cli/issues/1917
